src/com/lightdev/app/shtm/CombinedAttribute.java

```diff
@@ -238,9 +239,7 @@ public int getOppositeSide(final int side) {
     private void copyValues(final String[] v) {
         switch (v.length) {
             case 1:
-                for (int i = 0; i < values.length; i++) {
-                    values[i] = v[0];
-                }
+                Arrays.fill(values, v[0]);
```

Comment:
Using Arrays.fill for length 1 looks strange

Reply:
you're right, it was just to hide the for loop. I guess we can replace the for loop with  `values[0] = v[0];`